### PR TITLE
Publish QR code poster template for usage in event checkins via mobile device

### PR DESCRIPTION
**Pull request by EugenM-SAP**
## Checklist

* [ ] Test your changes as thoroughly as possible before you commit them. Preferably, automate your test by unit/integration tests.
* [ ] Make sure `mvn -P integration-tests clean verify` runs for the whole project and, if you touched any code for a service in the `services` folder, ensure it can be run with `spring-boot:run`

### CODE REVIEW

File: services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/qrcode/QrCodeTemplateLoader.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    return loadPosterTemplate("pt-ios-poster-1.0.0.pdf");
+  }
+
+  private ByteString loadPosterTemplate(String filename) {
+    try {
+      return ByteString.readFrom(this.getClass().getClassLoader().getResourceAsStream(filename));
+    } catch (IOException e) {
+      logger.error(
+          "Could not load '" + filename + "' QR poster template from the application package", e);
+      //At the moment just log the error but do not interfere with normal application startup
```
Comment (hilmarf) [inline services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/qrcode/QrCodeTemplateLoader.java:28]: follow up needed

File: services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/qrcode/QrCodeTemplateLoader.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+public class QrCodeTemplateLoader {
+
+  private static final Logger logger = LoggerFactory.getLogger(QrCodeTemplateLoader.class);
+
+  public ByteString loadAndroidTemplateAsBytes() {
+    return loadPosterTemplate("pt-android-poster-1.0.0.xml");
+  }
+
+  public ByteString loadIosTemplateAsBytes() {
+    return loadPosterTemplate("pt-ios-poster-1.0.0.pdf");
```
Comment (hilmarf) [inline services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/qrcode/QrCodeTemplateLoader.java:19]: please create follow up tasks to load from VAULT environment - but for now it's OK

File: services/distribution/src/main/resources/application.yaml [hunk 3]
```diff
@@ -161,9 +161,32 @@ services:
...
+      description-text-box:
+        offsetX: ${EVREG_QR_CODE_POSTER_TEMPLATE_IOS_DESCRIPTION_OFFSET_X:80}
+        offsetY: ${EVREG_QR_CODE_POSTER_TEMPLATE_IOS_DESCRIPTION_OFFSET_Y:510}
+        width: ${EVREG_QR_CODE_POSTER_TEMPLATE_IOS_DESCRIPTION_WIDTH:420}
+        height: ${EVREG_QR_CODE_POSTER_TEMPLATE_IOS_DESCRIPTION_HEIGHT:20}
+        fontSize: ${EVREG_QR_CODE_POSTER_TEMPLATE_IOS_DESCRIPTION_FONT_SIZE:14}
+        fontColor: ${EVREG_QR_CODE_POSTER_TEMPLATE_IOS_DESCRIPTION_FONT_COLOR:black}
+    android-qr-code-poster-template:
+      published-archive-name: qr_code_poster_template_android
+      template:
```
Comment (hilmarf) [inline services/distribution/src/main/resources/application.yaml:179]: does this work? empty value?

File: services/distribution/src/main/resources/application.yaml [hunk 4]
```diff
@@ -161,9 +161,32 @@ services:
         require-cts-profile-match: ${PPA_PPAC_REQUIRE_CTS_PROFILE_MATCH:false}
         require-evaluation-type-basic: ${PPA_PPAC_REQUIRE_EVALUATION_TYPE_BASIC:false}
         require-evaluation-type-hardware-backed: ${PPA_PPAC_REQUIRE_EVALUATION_TYPE_HARDWARE_BACKED:false}
-
-
-
+    ios-qr-code-poster-template:
+      published-archive-name: qr_code_poster_template_ios
+      template:
```
[the remark this answers is not in the document]
Comment (EugenM-SAP) [reply services/distribution/src/main/resources/application.yaml:166]: Yes

File: services/distribution/src/main/resources/application.yaml (hunk 3 above)
Comment (EugenM-SAP) [reply services/distribution/src/main/resources/application.yaml:179]: yep

**Review by hilmarf — APPROVED** [review]

**Review by roesslerj — APPROVED** [review]

**Pull request merged**